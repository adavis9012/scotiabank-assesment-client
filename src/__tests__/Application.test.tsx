import React from 'react';
import { shallow } from 'enzyme';
import Application from '../Application';

test('renders the component', () => {
  const component = shallow(<Application />);

  expect(component).toMatchSnapshot();
});