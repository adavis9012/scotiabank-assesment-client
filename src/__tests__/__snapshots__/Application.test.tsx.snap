// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the component 1`] = `
<div
  className="application"
>
  <Header />
  <main
    className="application_wrapper"
  >
    <Route
      component={
        Object {
          "$$typeof": Symbol(react.memo),
          "compare": null,
          "type": [Function],
        }
      }
      exact={true}
      path="/"
    />
    <Route
      component={
        Object {
          "$$typeof": Symbol(react.memo),
          "compare": null,
          "type": [Function],
        }
      }
      exact={true}
      path="/accounts"
    />
    <Route
      component={
        Object {
          "$$typeof": Symbol(react.memo),
          "compare": null,
          "type": [Function],
        }
      }
      exact={true}
      path="/movements/:id"
    />
  </main>
</div>
`;
